core/src/program.rs

```diff
@@ -98,7 +98,7 @@ impl ProgramId {
 #[derive(Clone, Debug, Decode, Encode)]
 pub struct Program {
     id: ProgramId,
-    code: Vec<u8>,
+    code_with_meta: CodeWithMetadata,
```

Comment:
Why does it have to be `CodeWithMeta` and not just wasm bytes?

Reply:
Will answer in discord and paste link. It requires some discussions.

The reason for that is actually a decision to store code metadata for codes, which are saved from `submit_program` extrinsic. More precisely, from `process_queue`.

Reply:
https://discord.com/channels/891063355526217738/905188682745655317/919139083484291093